engine/classes/Elgg/I18n/Translator.php

```diff
@@ -18,6 +18,15 @@ class Translator {
 	private $CONFIG;
 
 	/**
+	 * Regexp for finding keyword arguments in language strings.
+	 * Accepts forms like {label}, { label}, and {label } where label can include
+	 * a-z, A-Z, 0-9, _, -, :, /, and .
```

Comment:
Flexibility. Our project was using a mix of separating chars: `notifications:group-warning`, `notifications/groups/info` If we're still wanting to roll our version and not adopt ext-intl, we can probably shorten this to "everything that isn't a closing tag."